README.md

```diff
@@ -1,5 +1,35 @@
-# Firebase Functions Python SDK
+# Cloud Functions for Firebase Python SDK
 
-Welcome GitHub lurker! This repository is a work in progress. It will eventually let users create Cloud Functions for Firebase in Python. This feature is under very early development and not ready for testing. When it is ready, you can get first access by signing up for the [Firebase Alpha Program](https://services.google.cn/fb/forms/firebasealphaprogram/).
+The [`firebase-functions`](https://pypi.org/project/firebase-functions/) package provides an SDK for defining Cloud Functions for Firebase in Python.
 
-While you wait for us to get this feature ready for the public, we'd really appreciate you not blowing this up. Let's keep things on the down low for now. 🤫
+Cloud Functions provides hosted, private, and scalable environment where you can run server code. The Firebase SDK for Cloud Functions integrates the Firebase platform by letting you write code that responds to events and invokes functionality exposed by other Firebase features.
+
+## Learn more
+
+Learn more about the Firebase SDK for Cloud Functions in the [Firebase documentation](https://firebase.google.com/docs/functions/) or [check out our samples](https://github.com/firebase/functions-samples).
+
+Here are some resources to get help:
+
+- Start with the quickstart: https://firebase.google.com/docs/functions/write-firebase-functions
+- Go through the guide: https://firebase.google.com/docs/functions/
+- Read the full API reference: https://firebase.google.com/docs/reference/functions/
```

Comment:
This can link to directly to Python - I believe it's going to be `https://firebase.google.com/docs/reference/functions/2nd-gen/python` but @egilmorez can confirm

Reply:
That would work!  Staged at

https://firebase.devsite.corp.google.com/docs/reference/functions/2nd-gen/python?db=egilmore-2ndGenCF3

with the relevant part of the link being

/docs/reference/functions/2nd-gen/python

Reply:
Oh also -- /write-firebase-functions does not exist (though it may redirect correctly).

Instead use /get-started

Reply:
Done!